monitoring/my-kube-prometheus/alertmanager-config.yaml

```diff
@@ -0,0 +1,30 @@
+# external alertmanager yaml
```

Comment:
is alert manager required? seems like we can just deploy without until configuring it to work with discord

Reply:
> 

It's the full stack. Prometheus operator, node exporter, grafana, prometheus, alert manager + some nice default dashboards and alerts

Reply:
Sure, we can skip it until we have the actual alerts